Resources/Prototypes/_CD/Entities/Objects/Devices/pda.yml

```diff
@@ -101,3 +102,17 @@
       state: pda-roboticist
     - type: RenameId
       value: cd-job-title-roboticist
+
+- type: entity
+  id: OffDutyPDA
+  parent: BasePDA
```

Comment:
If everything here is the same as the passenger PDA it might be better to just inherit from it instead of BasePDA.

Reply:
See no other PDA does this so I wasn't sure if I wanted to vary at all. (Also this might make more since I'm going to add the sprites now.)

Reply:
Yeah, if you add sprites there is basically no reason to do that.